web/styles/app_variables.css

```diff
@@ -1103,6 +1103,205 @@
     --background-color-active-dropdown-item: hsl(220deg 12% 4.9% / 5%);
     --background-color-active-typeahead-item: hsl(221.14deg 89.74% 92.35%);
     --color-typeahead-option-label: var(--grey-500);
+
+    /* BUTTONS */
```

Comment:
Let's call this heading "Action buttons", since that's what we're calling this generic category, in the code comments?

Reply:
Makes sense, I'll rename this.

Reply:
Renamed to "Action Buttons".